polyglot/upload.py

```diff
@@ -55,9 +55,11 @@ class Upload(bitsv.PrivateKey):
     """
     A simple interface to a multitude of bitcoin protocols
     """
-    def __init__(self, wif=None, network='main'):
+    def __init__(self, wif=None, network='main', fee=None, utxo_min_confirmations=1):
```

Comment:
I think fee=None is wrong and should be fee=1 (satoshi/byte) because trying to do arithemetic operations with None type will error...

Do you agree?